src/test/java/seedu/address/model/person/UniquePersonListTest.java

```diff
@@ -40,11 +39,11 @@ public void contains_personInList_returnsTrue() {
     }
 
     @Test
-    public void contains_personWithSameIdentityFieldsInList_returnsTrue() {
-        uniquePersonList.add(ALICE);
-        Person editedAlice = new PersonBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
+    public void contains_personWithSameAddressInList_returnsTrue() {
```

Comment:
Perhaps this method could be more aptly renamed as `contains_clientWithSameAddressInList_returnsTrue()`